### ISSUE TITLE
WPA2-EAP: attribute "Anonymous Identity" ignored

### ISSUE THREAD

**Issue by JacobKochems** [markusfisch/BinaryEye#231]
Hello, I installed the app version 1.45.0 within Android 8.1 and created a QR code with the following content:

`WIFI:T:WPA2-EAP;S:my-wifi;P:5abpa8mrwjmr35in;E:PEAP;I:ident;A:anonymous;PH2:MSCHAPV2;;`

It seems that the key-value pair `A:anonymous` is ignored or "gets lost" somehow since the corresponding field in the connection profile form is empty.

Thanks for this app by the way, it works great otherwise.

**Comment by markusfisch** [comment]
Implemented in https://github.com/markusfisch/BinaryEye/commit/867f179a371854c07711efd34051c6ac22efedcb

**Comment by markusfisch** [comment]
No, this isn't part of the ZXing barcode scanning library, but of the accompanying (and abandoned) Android app. So it's part of the repo but not of the library.

The implementation in Binary Eye was a contribution of @molikuner, and it was re-implemented because Binary Eye is written in Kotlin while the Android app of ZXing was written in Java.
